### ISSUE TITLE
Add structure to the experiment to test out different loss function

### ISSUE THREAD

**Issue by vdakov** [vdakov/encoder-decoder-change-detection#2]
Currently the experiment uses a fixed negative log likelihood losss function with weights per positive and negative class that scale based on the number of black/white pixels inside it. This is an imperfect setup prone to overfitting. Extend the experiment to allow other loss functions.
